src/test/java/Homework21Test.java

```diff
@@ -0,0 +1,44 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework21Test extends BaseTest {
+    Actions actions;
+    actions = new Actions(driver);
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/212632060-ccce4e3a-387e-4fb3-8179-39730707c6c7.png)

Solution

```
  String playlistnewname = "Jarrod Playlist";

    @Test
    public void renamePlaylist() {
        logIn("skyeman75@gmail.com", "te$t$tudent");
        verifyLogIn();
        contextClickSelectPlaylist();
        contextClickEdit();
        Assert.assertTrue(checkPlaylistRenamed().isDisplayed());
    }

    public void contextClickSelectPlaylist() {
        Actions actions = new Actions(driver);
        wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector(".playlist:nth-child(3)"))).click();
        WebElement playListName = driver.findElement(By.cssSelector(".playlist:nth-child(3)"));
        actions.contextClick(playListName).perform();
    }

    public void contextClickEdit() {
        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("li[data-testid^='playlist-context-menu-edit'"))).click();
        WebElement editField = driver.findElement(By.cssSelector("input{name='name']"));
        editField.sendKeys(playlistnewname);
        editField.sendKeys(Keys.ENTER);
    }

    public WebElement checkPlaylistRenamed() {
        return driver.findElement(By.cssSelector("div.success.show"));
    }
```